src/ListView/DeletePageModal.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from "react";
+import { Modal } from "@sailthru/stui-components";
+import { deletePage } from "../common/pagesApi";
+
+const PageSuccessDeleteMessage = "Page successfully deleted.";
+const PageErrorDeleteMessage =
+  "Page was unable to be deleted, please try again later.";
+
+function DeletePageModal({
+  pageId,
+  pageName,
+  setDisplayDeleteModal,
+  setDisplayNotification,
```

Comment:
nit-pick: The other notification variables follow the pattern `setNotification<Variable>`
```suggestion
  setNotificationDisplay,
```